packages/api-client/src/views/Request/RequestSidebar.vue

```diff
@@ -183,14 +186,35 @@ onBeforeUnmount(() => {
       </div>
     </template>
     <template #button>
-      <SidebarButton
-        v-if="!isReadonly"
+      <div
+        class="relative z-10 pt-0 md:px-2.5 md:pb-2.5 -translate-y-full w-[inherit]"
         :class="{
           'empty-sidebar-item': activeWorkspaceRequests.length === 1,
-        }"
-        :click="commandPaletteBus.emit">
-        <template #title>Add Item</template>
-      </SidebarButton>
+          'border-t-1/2': activeWorkspaceRequests.length === 1,
```

Comment:
should we set it to be less or equal to one here? currrently if we remove the all the request then the state is not displayed anymore

Reply:
sure! great idea